CHANGELOG-1.3.md

```diff
@@ -1,9 +0,0 @@
-# Changelog since v1.2.0
-
-## Breaking Changes
-
-- Add a prefix "external-snapshotter-leader" and the driver name to the snapshotter leader election lock name. Rolling update will not work if leader election is enabled because the lock name is changed in v1.3.0. ([#129](https://github.com/kubernetes-csi/external-snapshotter/pull/129), [@zhucan](https://github.com/zhucan))
-
-## Other Notable Changes
-
-- Snapshotter will no longer call ListSnapshots if the CSI Driver does not support this operation. ([#138](https://github.com/kubernetes-csi/external-snapshotter/pull/138), [@ggriffiths](https://github.com/ggriffiths))
```

Comment:
Also this one?

Reply:
This one is already backported to 1.2.1: https://github.com/kubernetes-csi/external-snapshotter/blob/release-1.2/CHANGELOG-1.2.md#notable-changes-1

Reply:
We should still include it here since it's not in 1.2.0?

Reply:
ok.  I don't know that we should include things that are already backported.  let me check if there are other things missing.

Reply:
Yeah since this is the first new minor release, we include everything since 1.2.0

Reply:
Changed start-sha to the commit for 1.2.0 and re-run release-notes.  Now I should have everything since then.